proposals/nnnn-AOA-multiplexing.md

```diff
@@ -0,0 +1,53 @@
+# AOA multiplexing similar to Android BT/SPP multiplexing
+
+* Proposal: [SDL-NNNN](NNNN-AOA-multiplexing.md)
+* Author: [Jennifer Hodges](https://github.com/jhodges55)
+* Status: Awaiting review
+* Impacted Platforms: [ Android ]
+
+
+## Introduction
+
+Add "multiplexing" to AOA transport. ![Overview](../assets/proposals/nnnn-AOA-multiplexing/nnnn-AOA-multiplexing_overview.png)
+This change introduces a better user experience as followings:
+- Multi navigation apps to be able to exchanged by user.
```

Comment:
Should this be "Multiple navigation apps can be exchanged by user."?  I'm not sure what's intended here.